src/coreclr/tools/Common/JitInterface/CorInfoInstructionSet.cs

```diff
@@ -768,6 +768,59 @@ public InstructionSetInfo(string name, string managedName, InstructionSet instru
             }
         }
 
+        public static IEnumerable<string> AllCpuFamilies()
```

Comment:
This file is auto-generated (see note at the top). This should live somewhere else (or be autogenerated).